File: PS_JAVA/DAY_3/Assighment_Solution/CreateFile.java
import java.io.File;

class CreateFile {
    public static void main(String[] args) {
        File file = new File("newfile.txt");
        try {
            boolean val = file.createNewFile();
            if (val) {
                System.out.println("New file created");
            } else {
                System.out.println("File already exists");
            }
        } 
        catch(Exception e) {
            e.getStackTrace();
        }
    }
}

File: PS/Week_2/DAY_2/CreateFile.scala
import java.io._

object Createfile {
    def main(args: Array[String]) {

        val newFile = new File("SampleFile2.txt");
        val writeFile = new PrintWriter(newFile);
        writeFile.write("This text will be written in newfile.")
        writeFile.write("I have created this file. training file object name is pgm");
        writeFile.close();
    }
}